lib/models/material_colors.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+
+const MaterialColor accentColor =
+    const MaterialColor(0xFF969696, const <int, Color>{
+  50: Color.fromRGBO(150, 150, 150, .1),
+  100: Color.fromRGBO(150, 150, 150, .2),
+  200: Color.fromRGBO(150, 150, 150, .3),
+  300: Color.fromRGBO(150, 150, 150, .4),
+  400: Color.fromRGBO(150, 150, 150, .5),
+  500: Color.fromRGBO(150, 150, 150, .6),
+  600: Color.fromRGBO(150, 150, 150, .7),
+  700: Color.fromRGBO(150, 150, 150, .8),
+  800: Color.fromRGBO(150, 150, 150, .9),
+  900: Color.fromRGBO(150, 150, 150, 1),
+});
+
+const MaterialColor primarySwatchColor =
+    const MaterialColor(0xFF343539, const <int, Color>{
+  50: Color.fromRGBO(34, 35, 39, .1),
+  100: Color.fromRGBO(34, 35, 39, .2),
+  200: Color.fromRGBO(34, 35, 39, .3),
+  300: Color.fromRGBO(34, 35, 39, .4),
+  400: Color.fromRGBO(34, 35, 39, .5),
+  500: Color.fromRGBO(34, 35, 39, .6),
+  600: Color.fromRGBO(34, 35, 39, .7),
+  700: Color.fromRGBO(34, 35, 39, .8),
+  800: Color.fromRGBO(34, 35, 39, .9),
+  900: Color.fromRGBO(34, 35, 39, 1),
```

Comment:
po co tyle rodzajów? zakładam, że to jakiś weight jest.

Reply:
trzeba tak, jak wszystkich odcieni nie podasz od .1 do 1 to flutter wywala błąd